aten/src/ATen/native/cuda/MultinomialKernel.cu

```diff
@@ -43,7 +43,11 @@ inline __device__ bool _isinf(T x) {
   }
 }
 
+#ifdef USE_ROCM
+#define MAX_NUM_BLOCKS 64
```

Comment:
Why was this not needed before?

Reply:
Hm, not sure.  Let's remove  this and see what happens with CI.